packages/design/src/animation.mdx

```diff
@@ -5,10 +5,12 @@ route: /animation
 ---
 
 # Animation
+
 ## Patterns
 
 Animations in Jobber fall under categories based on their purpose:
 
+
 ###Reveal
```

Comment:
Still need spaces between the ### and the text, like:

`###Reveal` needs to be `### Reveal`